src/Directory.Build.props

```diff
@@ -1,3 +1,6 @@
 <Project>
   <Import Project="$([MSBuild]::GetPathOfFileAbove('FSharp.Directory.Build.props', '$(MSBuildThisFileDirectory)../'))" />
+  <PropertyGroup>
```

Comment:
@brettfo can correct me if I'm wrong, but I believe the correct place for this is:

https://github.com/Microsoft/visualfsharp/blob/a7913e4062f97cb8fa858786d0755fa9d8bde34e/FSharpBuild.Directory.Build.props

or
https://github.com/Microsoft/visualfsharp/blob/master/FSharpBuild.Directory.Build.targets

Reply:
That would apply it to all projects wouldn't it?  I don't think all projects in `vsintegration` are free of warnings